oneview_redfish_toolkit/tests/api/test_capability.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+
+from oneview_redfish_toolkit.api.capability import Capability
+from oneview_redfish_toolkit.tests.base_test import BaseTest
+
+
+class TestCapability(BaseTest):
+    """Tests for Capability class"""
+
+    with open('oneview_redfish_toolkit/mockups/oneview/'
+              'ServerProfileTemplate.json') as f:
+        server_profile_template = json.load(f)
+
+    with open('oneview_redfish_toolkit/mockups/redfish/Capability.json') as f:
+        capability_mockup = json.load(f)
+
+    def test_class_instantiation(self):
+        # Tests if class is correctly instantiated and validated
+        try:
+            capability = Capability(self.server_profile_template)
+        except Exception as e:
```

Comment:
This exception handler is unnecessary, because if class instantiation fail, the test will raise a exception and fail showing the error.
But no problem to me keep this code, just aggregates bit worth.

Reply:
Done.